tests/base/cache_pool_test.py

```diff
@@ -525,6 +530,23 @@ def test_value_wrong_type(
         assert "Expecting <class 'tests.base.cache_pool_test.Bar'> got Foo" in str(e)
 
 
+def test_value_wrong_key_result(
+    memcache_socket: MemcacheSocket, cache_pool: FakeCachePool
+) -> None:
+    memcache_socket.get_response.return_value = Value(
+        size=3,
+        int_flags={
+            IntFlag.CLIENT_FLAG: 0,
+        },
+        token_flags={TokenFlag.KEY: b"NOT_FOO"},
+    )
+    try:
+        cache_pool.get(key=Key("foo"))
+        raise AssertionError("Should not be reached")
```

Comment:
This could also be `pytest.fail("message")`

Reply:
TIL